subsys/bluetooth/audio/has_client.c

```diff
@@ -50,6 +52,18 @@ static struct has_inst {
 	struct bt_gatt_subscribe_params active_index_subscription;
 } has_insts[CONFIG_BT_MAX_CONN];
 
+/** HAS control point procedure */
+struct has_cp_req {
+	/** User data to be passed to the callback */
+	void *user_data;
+
+	/** GATT procedure parameters */
+	struct bt_gatt_write_params params;
+};
+
+K_MEM_SLAB_DEFINE_STATIC(req_slab, sizeof(struct has_cp_req), CONFIG_BT_L2CAP_TX_BUF_COUNT,
+			 __alignof__(struct has_cp_req));
```

Comment:
You know, I'll remove this. I thought it would be nice to handle multiple requests to make use of EATT as well as be able to set the active preset when Read Preset operation is in progress, but to reduce the memory consumption, tel's handle one request at the time